elektrycznosc_statystyki_kk.sql

```diff
@@ -0,0 +1,356 @@
+/*
+TEMAT: Analiza zużycia / produkcji energii elektrycznej.
+	
+Zakres lat: 1960 - 2012
+**/
+--=============================================================================
+-- Sprawdzenie kodów związanych z elektrycznością
+
+-- jakie kody związane z elektrycznością
+select * from indicators i
+where lower(indicatorname) like '%electr%'
+order by indicatorname ;
+
+-- jakie kody produkcji elektryczności
+select distinct indicatorname, indicatorcode from indicators i
+where lower(indicatorname) like '%electricity prod%'
+order by indicatorcode;
+
+-- jakie kody konsumpcji elektryczności
+select distinct indicatorname, indicatorcode from indicators i
+where lower(indicatorname) like '%electric power cons%'
+order by indicatorcode;
+
+/*
+ Zakres wskaźników:
+	Electricity production from coal sources (% of total)	EG.ELC.COAL.ZS
+	Electricity production from oil, gas and coal sources (% of total)	EG.ELC.FOSL.ZS
+	Electricity production from hydroelectric sources (% of total)	EG.ELC.HYRO.ZS
+	Electricity production from natural gas sources (% of total)	EG.ELC.NGAS.ZS
+	Electricity production from nuclear sources (% of total)	EG.ELC.NUCL.ZS
+	Electricity production from oil sources (% of total)	EG.ELC.PETR.ZS
+	Electricity production from renewable sources, excluding hydroelectric (kWh)	EG.ELC.RNWX.KH
+	Electricity production from renewable sources, excluding hydroelectric (% of total)	EG.ELC.RNWX.ZS
+*/
+--=============================================================================
+
+-- Sprawdzenie kto bierze udział w notowaniach
+select * from country c;
+
+-- Zauważam, że w rekordach ukrywają się zbiorcze statystyki World/ Europa itd.
+-- Można je zindentyfikować przez pole alpha2code, które zawiera wtedy cyfrę.
+-- Są też stowarzyszenia i frupy krajów:
+-- XC, EU, XE, XD, XR, XS, XJ, ZJ, XL XO, XM, XN, ZQ, XQ, XP, XU, OE,  ZG, ZF, XT
+-- niektóre dane w kWh inne w %
+
+--regiony
+select * from country c, 
+regexp_matches(alpha2code, '[0-9]');
+
+--kraje
+select * from country c
+where c.alpha2code !~ '[%0-9%]' 
+		and c.alpha2code !~'[X%]' 
```

Comment:
Nie wiem, czy to przypadkiem nie wina githuba, że tak to wyświetla, ale z jakiegoś powodu nagle kod "odjeżdza" Ci w prawą stronę. Po kontekście ogólnym i kodach, które wysyłałeś mi wcześniej na slacku domyślam się, że nie takie było zamierzenie - warto sprawdzić, czy da się to poprawić, żeby wyglądało lepiej i na githubie po zacommitowaniu, bo te nagłe wcięcia znacznie psują czytelność kodu

Reply:
To sprawa VS Code -> Github... Sprawdzę i poprawię.